<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>421</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя выходного файла</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>философия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>https://ru.wikipedia.org/wiki/%D0%A4%D0%B5%D0%B4%D0%BE%D0%BD_(%D0%9F%D0%BB%D0%B0%D1%82%D0%BE%D0%BD)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Запрос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="unsplittableCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Точный запрос</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>661</width>
     <height>311</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Параметры</string>
   </property>
   <widget class="QPushButton" name="setParamButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать параметры</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="linkCheck">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать по ссылкам на странице</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество ссылок</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="folCount">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>47</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="linkCount">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>77</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество синонимов для слова</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>47</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество выбираемых элементов, после найденного</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество выбираемых элементов, предшествующих найденному</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="synCheck">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Использовать поиск по синонимам</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="precCount">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>17</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="synCount">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>107</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="subThemeTwoRadButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Подтемам 2 уровня</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="semanticCheck">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Использовать базу знаний</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="themeRadButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Темам</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="subThemeOneRadButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Подтемам 1 уровня</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать по:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
